.babelrc

```diff
@@ -1,3 +1,21 @@
 {
-  "presets": ["es2015", "stage-2", "react"]
+  "presets": ["react"],
+  "env": {
+    "commonjs": {
+      "plugins": [
```

Comment:
Supposedly the env and non-env configs should merge, but I was seeing the below issue when I tried to consolidate. Any help is appreciated.

Tried this:
```js
{
  "presets": ["react"],
  "plugins": [
      "transform-class-properties", 
      "transform-object-rest-spread"
  ],
  "env": {
    "commonjs": {
      "presets": ["env"]
    },
    "es": {
      "presets": [
        ["env", { "modules": false }]
      ]
    }
  }
}
```

Got this:
![image](https://user-images.githubusercontent.com/2483304/27213064-184edac2-522a-11e7-8b1b-dd15d5219c41.png)